client/common/js/urls.js

```diff
@@ -1,141 +0,0 @@
-'use strict';
```

Comment:
Ah, I didn't know it had to be an individual commit for git to infer it was as rename, thanks for letting me know :)